apps/extension/client/src/documentationLinks.ts

```diff
@@ -1,46 +1,72 @@
-const baseUrl = "https://xstate.js.org/docs/guides";
+const baseUrl = "https://graph-docs.vercel.app";
 
 interface DocumentationLink {
   keyWords: string[];
   description: string;
   link: string;
 }
 
+// TODO: Can we create these links using the frontmatter from our docs in a build step?
 const documentationLinks: DocumentationLink[] = [
   {
     keyWords: ["actions"],
     description: "Actions",
-    link: `${baseUrl}/actions.html`,
+    link: `${baseUrl}/actions/actions`,
   },
   {
-    keyWords: ["context"],
+    keyWords: ["context", "initial"],
     description: "Context",
-    link: `${baseUrl}/context.html`,
+    link: `${baseUrl}/actions/context`,
   },
   {
-    keyWords: ["initial"],
-    description: "Initial Context",
-    link: `${baseUrl}/context.html#initial-context`,
+    keyWords: ["invoke", "services"],
+    description: "Services",
+    link: `${baseUrl}/services/intro`,
   },
   {
-    keyWords: ["events"],
+    keyWords: ["event", "events"],
     description: "Events",
-    link: `${baseUrl}/events.html`,
+    link: `${baseUrl}/basics/what-is-a-statechart#events`,
   },
   {
     keyWords: ["states"],
     description: "States",
-    link: `${baseUrl}/states.html`,
+    link: `${baseUrl}/basics/what-is-a-statechart#states`,
+  },
+  {
+    keyWords: ["on"],
+    description: "Transitions and events",
+    link: `${baseUrl}/what-is-a-statechart#transitions-and-events`,
+  },
+  {
+    keyWords: ["assign"],
+    description: "Updating context",
+    link: `${baseUrl}/actions/context#updating-context`,
   },
   {
     keyWords: ["tsTypes", "typegen"],
     description: "Typegen",
-    link: `${baseUrl}/typescript.html#typegen`,
+    link: `${baseUrl}/typescript/typegen`,
   },
   {
     keyWords: ["schema"],
```

Comment:
`schema` is tricky because there's no central spot where we talk about how to use - we split it up over several files. So what you want is to ensure we're in `schema.context` to go to the context section, `schema.events` to go to the events section, and `schema.services` to talk about invoked services with typegen.

Reply:
Thank you for the feedback; I have added some special handling for the schema object in the latest commit. But I need to figure out where to link to in the three cases:

1. schema.context -> https://graph-docs.vercel.app/actions/context#type-script
2. schema.events -> ?
3. schema.services -> ?

Can you help with the correct links?